### PR TITLE
Always surround up and down clauses in parentheses

### PR DESCRIPTION
**Pull request by ryanslade**
Closes https://github.com/xataio/pgroll/issues/473

### CODE REVIEW

File: pkg/migrations/op_add_column_test.go [hunk 1]
```diff
@@ -786,6 +786,88 @@ func TestAddColumnWithUpSql(t *testing.T) {
...
+							},
+						},
+					},
+				},
+				{
+					Name: "02_add_column",
+					Operations: migrations.Operations{
+						&migrations.OpAddColumn{
+							Table: "products",
+							Up:    "select name",
```
Comment (exekias) [inline pkg/migrations/op_add_column_test.go:817]: I believe we also want to test just `name` here? ie
```suggestion
							Up:    "name",
```

Comment (ryanslade) [reply pkg/migrations/op_add_column_test.go:817]: I'll add a separate test for that. The one with `select` definitely failed without `( )`

Comment (ryanslade) [reply pkg/migrations/op_add_column_test.go:817]: Done and looks fine

**Review by exekias — APPROVED** [review]

**Comment by exekias** [comment]
probably worth updating docs & examples?

**Review by andrew-farries — COMMENTED** [review]
I think this is fine. We should update the documentation and example migrations to remove explicit parens as part of this PR too I think.

**Pull request merged**